.editorconfig

```diff
@@ -13,8 +13,5 @@ insert_final_newline = true
 indent_style = space
 indent_size = 2
 
-[*.hbs]
-insert_final_newline = false
```

Comment:
Separate commit for this.

Reply:
It is line consistencybear fixes.

Reply:
What is it fixing?

Please create an issue explaining why this should be removed, and add a separate commit for closing that new issue.

Reply:
You are right it is unrelated, i will re add it.

Reply:
My request has not been done yet

Reply:
Done